Makefile

```diff
@@ -179,6 +179,12 @@ test-e2e: $(KUBECTL) $(GINKGO) $(ENVSUBST) set-flavor e2e-image
 		-e2e.use-existing-cluster=$(SKIP_CREATE_MGMT_CLUSTER) \
 		-e2e.flavor="$(E2E_FLAVOR)"
 
+# Basic checks for deploying kind cluster and required providers
+.PHONY: test-sanity
+test-sanity: $(ENVSUBST) set-flavor e2e-image
+	$(ENVSUBST) < $(E2E_CONF_FILE) > $(E2E_CONF_FILE_ENVSUBST)
+	go test -v ./test/e2e/suite_test.go -e2e.config="$(E2E_CONF_FILE_ENVSUBST)" -e2e.artifacts-folder="$(ARTIFACTS)" -ginkgo.v -ginkgo.trace -ginkgo.progress
+
```

Comment:
just wondering instead of creating this new target, can we reuse the `test-e2e` target with `GINKGO_FOCUS` set to some tag to run only the single testcase?

Reply:
agree, that would be easier. For now I set it to a non-existent tag since the required tests run as a part of the `BeforeSuite` and I couldn't find if there was a way to add a construct to it so that it matches the `GINKGO_FOCUS`. PTAL.